### PR TITLE
colabfold nf-test

### PR DESCRIPTION
**Pull request by vagkaratzas**

### CODE REVIEW

File: modules/ebi-metagenomics/colabfold/colabfoldbatch/main.nf [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    colabfold_batch \\
+        $args \\
+        ${fasta} \\
+        ${prefix}
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        colabfold: 1.5.2
```
Comment (mberacochea) [inline modules/ebi-metagenomics/colabfold/colabfoldbatch/main.nf:31]: There is a mismach, the container version is "1.5.5" and the version here is "1.5.2". Can't you get the version the good ol' way (colabfold_batch --version..)?

File: modules/ebi-metagenomics/colabfold/colabfoldbatch/tests/main.nf.test.snap [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                "1": [
+                    "versions.yml:md5,66e451357b0b69c85226e5ed06a2bf57"
+                ],
+                "out_dir": [
+                    [
+                        {
+                            "id": "test",
+                            "single_end": false
+                        },
+                        [
```
Comment (mberacochea) [inline modules/ebi-metagenomics/colabfold/colabfoldbatch/tests/main.nf.test.snap:33]: image

Comment (mberacochea) [reply modules/ebi-metagenomics/colabfold/colabfoldbatch/tests/main.nf.test.snap:33]: Empty?

File: tests/config/nf-test.config
Comment (mberacochea) [inline tests/config/nf-test.config:1]: @vagkaratzas are the changes on this file reflecting the latest on nf-core?

**Review by mberacochea — CHANGES_REQUESTED** [review]
Thanks @vagkaratzas . I left a few comments

File: tests/config/nf-test.config
Comment (vagkaratzas) [reply tests/config/nf-test.config:1]: Exactly

File: modules/ebi-metagenomics/colabfold/colabfoldbatch/tests/main.nf.test.snap (hunk 2 above)
Comment (vagkaratzas) [reply modules/ebi-metagenomics/colabfold/colabfoldbatch/tests/main.nf.test.snap:33]: It seems this is how nf-test snapshots empty folders (this is the stub)

File: modules/ebi-metagenomics/colabfold/colabfoldbatch/main.nf (hunk 1 above)
Comment (mberacochea) [reply modules/ebi-metagenomics/colabfold/colabfoldbatch/main.nf]: I thought that would be the case.

**Review by mberacochea — APPROVED** [review]
:shipit:

**Pull request merged**